pages/common/gource.md

```diff
@@ -0,0 +1,28 @@
+# gource
+
+> Renders an animated tree diagram of Git, SVN, Mercurial and Bazaar repositories.
+> It shows files and folders being created, modified or removed over time.
+
+- Run gource in a directory (assuming the command is run from the project's root directory):
+
+`gource {{path/to/repository}}`
+
+- Set the resolution:
```

Comment:
I'd say it makes more sense in the first example; readers will likely assume that the same behavior carries over to the next ones.

But now that you mention it: the note in the command above should be reworded to match the command better. For example: "Run gource in a directory (if it isn't the repository's root directory, the root is seeked up from there)", or "Run gource in a directory (assuming it's the repository's root directory)", or something like that.